apps/_pomodoro_app/realtime/server.js

```diff
@@ -1,16 +1,23 @@
 const http = require('http')
 const socketio = require('socket.io')
-const port = parseInt(process.env.PORT || '3001')
 const express = require('express')
-const { init } = require('./index')
+const port = parseInt(process.env.PORT || '3001')
+const { init } = require('./pomodoroSocketIOServer')
+const prepareBackApp = require('./pomodoroExpressBackend')
+
+//Deps, todo(toplenboren) make DI
+const ConsoleLogger = require('./application/ConsoleLogger')
+const SimpleAuthMiddleware = require('./application/SimpleAuthMiddleware')
+const StoreRepo = require('./store/store')
+const LinkGenerator = require('./application/LinkGenerator')
 
 async function prepareSocketIOServer(server) {
     const io = socketio(server)
-    init(io)
+    init(io, new ConsoleLogger(), SimpleAuthMiddleware, StoreRepo)
```

Comment:
why without new? it's better to use `simpleAuthMiddleware` if it's an object (not a constructor)

Reply:
Rewrote this